### PR TITLE
Starting from v1.19, remove specifying kubeedge version through `profile version`

### PR DESCRIPTION
**Pull request by luomengY**
…le vesrion

**What type of PR is this?**
Starting from v1.19, remove specifying kubeedge version through `profile version`

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by luomengY** [comment]
@Shelley-BaoYue  @WillardHu

File: keadm/cmd/keadm/app/cmd/helm/cloudcore.go [hunk 1]
```diff
@@ -90,21 +90,9 @@ func NewCloudCoreHelmTool(kubeConfig, kubeedgeVersion string) *CloudCoreHelmTool
 
 // Install uses helm client to install cloudcore release
 func (c *CloudCoreHelmTool) Install(opts *types.InitOptions) error {
-	externValueFile := ""
+	var externValueFile string
```
Comment (WillardHu) [inline keadm/cmd/keadm/app/cmd/helm/cloudcore.go:93]: The externValueFile variable seems to be of little use, and you can use opts.Profile directly.

Comment (luomengY) [reply keadm/cmd/keadm/app/cmd/helm/cloudcore.go]: ok

**Review by Shelley-BaoYue — COMMENTED** [review]
/lgtm

**Review by fisherxu — COMMENTED** [review]
/assign @WillardHu

**Comment by Shelley-BaoYue** [comment]
Please cherry-pick to relealse-1.19

**Comment by luomengY** [comment]
> Please cherry-pick to relealse-1.19

ok

**Pull request merged**